br/backup-and-restore-storages.md

```diff
@@ -84,14 +84,14 @@ This section provides some URI examples by using `external` as the `host` parame
 **Back up snapshot data to Azure Blob Storage**
 
 ```shell
-./br backup full -u "${PD_IP}:2379" \
+tiup br backup full -u "${PD_IP}:2379" \
 --storage "azure://external/backup-20220915?account-name=${account-name}&account-key=${account-key}"
 ```
 
 **Restore the `test` database from snapshot backup data in Azure Blob Storage**
 
 ```shell
-./br restore db --db test -u "${PD_IP}:2379" \
+tiup br restore db --db test -u "${PD_IP}:2379" \
 --storage "azure://external/backup-20220915account-name=${account-name}&account-key=${account-key}"
 ```
 
```

Comment:
@dveeden Could you please confirm if Line 131 needs to be adjusted to "tiup br backup full --pd "${PD_IP}:2379"
https://github.com/pingcap/docs/blob/29a6eda38422efdf8c9c1f104e6ee53eb377341e/br/backup-and-restore-storages.md?plain=1#L131